airlock/management/commands/run_file_uploader.py

```diff
@@ -0,0 +1,152 @@
+import logging
+import os
+import time
+
+from django.conf import settings
+from django.core.management.base import BaseCommand
+from opentelemetry import trace
+
+import old_api
+from airlock.business_logic import bll
+from airlock.enums import RequestStatus
+from airlock.users import User
+from services.tracing import instrument
+
+
+logger = logging.getLogger(__name__)
+
+# We need a user with the output-checker role to access some
+# bll methods
+system_user = User("system", output_checker=True)
+
+
+class Command(BaseCommand):
+    """
+    Poll database for approved requests that still have files pending upload
+    """
+
+    def add_arguments(self, parser):
+        # In production, we want this check to run forever. Using a
+        # function means that we can test it on a finite number of loops.
+        parser.add_argument("--run-fn", default=lambda: True)
+
+    def handle(self, *args, **options):
+        run_fn = options["run_fn"]
+
+        logger.debug("Watching for tasks")
+
+        while run_fn():  # pragma: no branch
```

Comment:
I think this may need a top level exception handler, to log and re-launch.

We don't know where an exception might come from. e.g. maybe bll.get_approved_requests() throws a sqlite locked error, which is a temporary error.

AIUI, in that case the management command will exit with an error.  And we would need to restart the entire container to restart it again.

I think this management command should do its best to stay running in the face of errors. Basically, it kind of needs to act as its own process supervisor (like gunicorn supervises the web workers, and restarts one if it errors).  Or else we may need to manually log in to L4 and restart the container if something goes wrong?